src/Microsoft.Azure.WebJobs.ServiceBus/EventHubs/EventHubAttribute.cs

```diff
@@ -10,8 +10,9 @@ namespace Microsoft.Azure.WebJobs.ServiceBus
     /// Setup an 'output' binding to an EventHub. This can be any output type compatible with an IAsyncCollector.
     /// </summary>
     [AttributeUsage(AttributeTargets.Parameter)]
+    [ConnectionProvider(typeof(ServiceBusAccountAttribute))]
```

Comment:
I was just thinking that applying SBAA and having it affect EventHub bindings might be confusing for people.